python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -1009,9 +1061,9 @@ def deblend(self, mExposure, catalog):
                 t0 = time.monotonic()
                 # Build the parameter lists with the same ordering
                 if self.config.version == "scarlet":
-                    blend, skipped = deblend(mExposure, foot, self.config, spectrumInit)
+                    blend, skippedSources, skippedBands = deblend(mExposure, foot, self.config, spectrumInit)
```

Comment:
Continuing the thought from above, you could have `deblend` just return `blend` and `skippedSources`, then set `skippedBands = []` here, so that the `deblend` outputs aren't forced to match `deblend_lite`.

Reply:
Yeah, that does make more sense.